tests/functional/TranslationRequestsApiFunctionalTest.php

```diff
@@ -158,7 +159,9 @@ public function testSearchTranslationRequests()
         self::assertTrue(0 === \count($items));
 
         $searchResponse = $this->translationRequestsApi->searchTranslationRequests(self::BUCKET_NAME,
-            (new SearchTranslationRequestParams())->setFileUri(\vsprintf('/posts/hello-world_1_%s_post.xml', [$time]))
+            (new SearchTranslationRequestParams())
+                ->setFileUri(\vsprintf('/posts/hello-world_1_%s_post.xml', [$time]))
+                ->setOriginalAssetKey([$originalAssetKey])
```

Comment:
Functional test has never caught such an issue because it didn't use array/object filters, just simple string filter.